cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -25,13 +25,12 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Masterminds/semver"
 	appRepov1 "github.com/kubeapps/kubeapps/cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1"
 	"github.com/kubeapps/kubeapps/cmd/assetsvc/pkg/utils"
 	"github.com/kubeapps/kubeapps/cmd/kubeapps-apis/core"
 	corev1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/packages/v1alpha1"
-	"github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/helm/packages/v1alpha1"
 	helmv1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/helm/packages/v1alpha1"
```

Comment:
sorry which line are you referring to?

Reply:
I guess it was:
`helmv1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/helm/packages/v1alpha1"`

`helmv1` seems to be referring to helm, rather than our plugin

Reply:
eeh, probably best to take that one to @absoludity. I didn't change any of that